### PR TITLE
672 - Change KTH-login from CAS to new system

### PR DESCRIPTION
**Pull request by arifjehoh**
# Prerequisite
For local testing test out with Docker for easy testing.
1. Create **Dockerfile** file in your directory.
```Docker
FROM python:3

ENV PYTHONUNBUFFERED 1

RUN apt-get update -y
RUN apt-get -y install binutils libproj-dev gdal-bin

RUN mkdir /ais
WORKDIR /ais/

ADD requirements.txt /ais/
RUN pip install -r requirements.txt

ADD . /ais/

```
2. Create **docker-compose.yml** file in your directory.
```docker
version: "3.3"

services:
  db:
    image: postgis/postgis
    environment:
      - POSTGRES_DB=ais_dev
      - POSTGRES_USER=ais_dev
      - POSTGRES_PASSWORD=ais_dev

  ais:
    build: .
    command: python manage.py runserver localhost:8080 --settings local_settings
    volumes:
      - .:/ais
    ports:
      - "8080:8080"
    depends_on:
      - db

```

3. Run in terminal `docker-compose up --build` and it will let you run the server.
4. [Optional] You might need to do changes in `manage.py`. Where it says `os.environ.setdefault("DJANGO_SETTINGS_MODULE", "ais.production.settings")` change it to `os.environ.setdefault("DJANGO_SETTINGS_MODULE", "ais.local.settings")` 
([For more instruction about setting up docker](https://github.com/armada-ths/ais/pull/673))
# Documentation
[Here you can read about the things changed.](https://github.com/armada-ths/ais/wiki/Code-documentation)

### CODE REVIEW

File: kth_login/views.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	token = oauth.kth.authorize_access_token(request)
+	user = oauth.kth.parse_id_token(request, token)
+	request.session['user'] = user
+	kth_id = user['kthid']
+
+	# Steps above (2 & 3)
+	callback(request, kth_id)
+	print(user)
+	#user = authenticate(user=request.session['user'])
+	#print(user)
```
Comment (arifjehoh) [inline kth_login/views.py:41]: Delete this before approving

File: kth_login/views.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+	return redirect('/')
+
+
+def callback(request, kth_id):
+	# kth_id = tree[0][0].text
```
Comment (arifjehoh) [inline kth_login/views.py:47]: Delete this. copied from old file. This used to extract the kth_id from the the parameter

**Review by arifjehoh — COMMENTED** [review]
Remove this comments before approving the PR. The comment in the kth_login/views.py authorize method, is only there for testing purpose.

File: kth_login/views.py (hunk 1 above)
Comment (arifjehoh) [reply kth_login/views.py:41]: @rrudling These comment should be removed, they are only there for testing. It's up to you if you want to remove it or not.

File: kth_login/views.py (hunk 2 above)
Comment (arifjehoh) [reply kth_login/views.py:47]: @rrudling These comment should be removed.

**Review by rrudling — APPROVED** [review]
**GOOD JOB! (10/10)**

**Pull request merged**